seminars.html

```diff
@@ -1,12 +1,13 @@
 ---
-title: Seminars
+title: All Seminars
 order: 2
 permalink: /seminars/
 layout: seminar
 match_collection: seminars
 ---
 
 <div id="papers">
+  <p>Papers and presentations that were evaluated as deserving a distinction are marked with &#x2B50.</p>
```

Comment:
```suggestion
  <p>Papers and presentations that were deemed by the faculty as worthy of distinction are marked with &#x2B50.</p>
```

This makes the language a little more consistent with the description in the course syllabus/Canvas materials.

Reply:
Could I ask that this be a separate PR that changes it in the individual seminar template as well then?

Reply:
Sure, we could do that. I hadn't noticed that this language was copied from elsewhere on the site.